### PR TITLE
Fix 032 launch review issues

### PR DESCRIPTION
**Pull request by LucasXu0**
### Feature Preview

closes https://github.com/AppFlowy-IO/AppFlowy/issues/3499

#### Add padding and reset to default button
Screenshot 2023-09-22 at 16 03 34

#### keep the previous font family
https://github.com/AppFlowy-IO/AppFlowy/assets/11863087/a9690876-474b-4f03-afe6-e381fe12315b

#### only show the text direction items on RTL mode
Screenshot 2023-09-22 at 15 50 51
Screenshot 2023-09-22 at 15 50 51

---

#### PR Checklist

- [ ] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart [hunk 1]
```diff
@@ -153,6 +155,22 @@ class _AppFlowyEditorPageState extends State<AppFlowyEditorPage> {
...
+    // only show the rtl item when the layout direction is ltr.
+    for (final item in textDirectionItems) {
+      if (isRTL) {
+        if (toolbarItems.every((element) => element.id != item.id)) {
+          toolbarItems.add(item);
+        }
+      } else {
+        toolbarItems.removeWhere((element) => element.id == item.id);
+      }
+    }
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart:172]: Maybe something like:
```suggestion
  _setRTLToolbarItems(isRL);
```

And then a method like this:
```
  void _setRTLToolbarItems(bool isRTL) {
    toolbarItems.removeWhere((item) => item.id.contains('text_direction_'));

    if (isRTL) {
      toolbarItems.addAll(textDirectionItems);
    }
  }
```
Additionally abstract this `'text_direction_'` into a `const textDirectionPrefix = "text_direction_";`

Comment (Xazin) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart:172]: And an integration test for this logic

**Review by richardshiue — APPROVED** [review]

File: frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart (hunk 1 above)
Comment (LucasXu0) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart:172]: I prefer to filter the text direction items with their IDs.

```dart
  void _setRTLToolbarItems(bool isRTL) {
    final textDirectionItemIds = textDirectionItems.map((e) => e.id);
    // clear all the text direction items
    toolbarItems.removeWhere(
      (item) => textDirectionItemIds.contains(item.id),
    );
    // only show the rtl item when the layout direction is ltr.
    if (isRTL) {
      toolbarItems.addAll(textDirectionItems);
    }
  }
```

Comment (LucasXu0) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart]: > And an integration test for this logic

Added.

**Pull request merged**